app/src/main/java/com/eis/geoCalendar/gps/GPSPosition.java

```diff
@@ -45,8 +49,16 @@ public double getLongitude() {
      */
     public float getDistance(@NonNull final GPSPosition otherPosition) {
         float[] results = new float[1];
-
         Location.distanceBetween(getLatitude(), getLongitude(), otherPosition.getLatitude(), otherPosition.getLongitude(), results);
         return results[0];
     }
+
+    @Override
+    public boolean equals(@Nullable Object obj) {
+        if(!(obj instanceof GPSPosition))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['if' is not followed by whitespace.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4822079)